libafl/src/executors/hooks/windows.rs

```diff
@@ -193,6 +205,12 @@ pub mod windows_exception_handler {
         panic::set_hook(Box::new(move |panic_info| unsafe {
             let data = &raw mut GLOBAL_STATE;
             let in_handler = (*data).set_in_handler(true);
+
+            assert!(
```

Comment:
An assert in a panic hook makes no sense, it'll just re-panic and result in an endless loop(?)